pyseir/cli.py

```diff
@@ -106,8 +109,9 @@ def _run_all(state=None, run_mode='default', generate_reports=True, output_inter
         cache_all_data()
 
     if state:
-        if not states_only:
-            _impute_start_dates(state.title())
+        # Deprecate temporarily since not needed.
```

Comment:
Why is this deprecated? Would be helpful for people like me with a bit more context